internal/resource/obcluster/obcluster_task.go

```diff
@@ -1182,7 +1182,7 @@ func AdjustParameters(m *OBClusterManager) tasktypes.TaskError {
 		}
 	}
 
-	if newResource.Cpu.Value() > 16 && oldResource.Cpu.Cmp(newResource.Cpu) != 0 {
+	if oldResource.Cpu.Value() > 16 && oldResource.Cpu.Cmp(newResource.Cpu) != 0 {
 		foundCpuCount := false
 		for i, p := range copiedCluster.Spec.Parameters {
 			if p.Name == "cpu_count" {
```

Comment:
calculate cpu count first, if newResource.Cpu is less than 16, set it to 16, and oldResource.Cpu.Value() > 16 is not necessary.

Reply:
OK, you are right. I got confused here hahah.